strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/BaseArtifactController.java

```diff
@@ -54,5 +55,9 @@ else if (request.getMethod().equals(RequestMethod.HEAD.name()))
 
         return true;
     }
+    public ResponseEntity<String> checkRepositoryAccess(){
+	
+	    return new ResponseEntity<>("success", HttpStatus.OK);
+    }
```

Comment:
Please, follow our [coding convention](https://strongbox.github.io/developer-guide/coding-convention.html):
```suggestion
    public ResponseEntity<String> checkRepositoryAccess()
    {
        return new ResponseEntity<>("success", HttpStatus.OK);
    }
```